### ISSUE TITLE
Improve accessibility & placement of flash banner

### ISSUE THREAD

**Issue by mjgiarlo** [sul-dlss/purl#1079]
Work on https://github.com/sul-dlss/purl/pull/1078 revealed that the accessibility and placement of the flash banner in purl could be improved. Example:

![Screenshot from 2024-07-24 14-06-04](https://github.com/user-attachments/assets/b44111dd-e400-487f-8f27-06eb6177da9b)

For the flash banner, we would like to:

- [ ] Increase the contrast between the background and the text color. @astridu's advice is to change the color of the banner text and the `X`-to-dismiss to black. (For the `X`, it is already black but is currently set to 50% opacity, so consider tweaking the opacity percentages?)
- [ ] Move the banner down a bit so that there is some space between the header and the banner.

This likely involves changes to or with the component library which lives in `public/sul_components/`. Consider switching purl to use the banner from the component library.

There are only a few uses of flash in the app, for what it's worth.

**Comment by astridu** [comment]
If we use the banners in the component library, then those should be accessible.
